### PR TITLE
feat(github): convert to typescript

### PR DESCRIPTION
**Pull request by viceice**
This pr converts the github platform to typescript.

### CODE REVIEW

File: lib/platform/github/index.ts [hunk 1]
```diff
@@ -169,22 +177,22 @@ async function initRepo({
   config.renovateUsername = renovateUsername;
   config.localDir = localDir;
   config.repository = repository;
-  [config.repositoryOwner, config.repositoryName] = repository.split('/');
+  [config.repositoryOwner, config.repositoryOwner] = repository.split('/');
```
Comment (rarkins) [inline lib/platform/github/index.ts]: This doesn't seem right?

Comment (ViceIce) [reply lib/platform/github/index.ts]: Ups. Schuld be reverted

Comment (ViceIce) [reply lib/platform/github/index.ts]: Fixed

**Comment by ViceIce** [comment]
just rebased to master

**Pull request merged**